tools/PsesLogAnalyzer/PsesLogAnalyzer.psd1

```diff
@@ -0,0 +1,129 @@
+#
+# Module manifest for module 'Pses-LogAnalyzer'
+#
+# Generated by: Keith
+#
+# Generated on: 11/23/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'PsesLogAnalyzer.psm1'
+
+# Version number of this module.
+ModuleVersion = '1.0.0'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = '99116548-ad0f-4087-a425-7edab3aa9e57'
+
+# Author of this module
+Author = 'Microsoft'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft'
+
+# Copyright statement for this module
+Copyright = '(c) 2017 Microsoft. All rights reserved.'
+
+# Description of the functionality provided by this module
+# Description = ''
+
+# Minimum version of the PowerShell engine required by this module
+# PowerShellVersion = ''
```

Comment:
Given class usage, this should be "5.1" I'm thinking

Reply:
We got class support in 5.0, right?

Reply:
I set it to 5.1.